system/Database/SQLite3/Forge.php

```diff
@@ -110,6 +110,31 @@ public function dropDatabase(string $dbName): bool
         return true;
     }
 
+    /**
+     * @param list<string>|string $columnNames
+     *
+     * @throws DatabaseException
+     */
+    public function dropColumn(string $table, $columnNames): bool
+    {
+        $columns = is_array($columnNames) ? $columnNames : array_map(trim(...), explode(',', $columnNames));
+        $result  = (new Table($this->db, $this))
+            ->fromTable($this->db->DBPrefix . $table)
+            ->dropColumn($columns)
+            ->run();
+
+        if (! $result && $this->db->DBDebug) {
+            throw new DatabaseException(sprintf(
```

Comment:
Shouldn't throwing an exception be moved to `Table::run()` method? Right now only `dropColumn()` will be handled this way and other operations not.

Reply:
I keep forgetting how specific the Forge class is in SQLite3.

It looks like we can't prepare any generic error message that will fit all `Table::run()` use cases. I think we can leave it as is.

Reply:
When something goes wrong, we also throw but on a `BaseConnection::query()` level.